Adding a translation in Indonesian language to the documentation

fritssasia opened the pull request:
Hello I am frits sasia, this is the result of my contribution. I added a translation for documentation in Indonesian language
I hope this can help and make it easier for people who are not fluent in English

The changes I made :
1. I created a `doc-translation` folder
2. I created a `README-IND.md` file which contains a translation of this documentation in Indonesian language
3. I edited the README.md original documentation file by adding a dropdown menu which lists the languages available for this documentation (allowing users to read documentation in other languages) and added info about the `doc-translation` folder in the `project structure` section.

I will try to invite my friends, maybe someone is interested for contributing here
I hope you like it, thank you :)

ref.: #36

fritssasia committed:
add documentation translation (indonesian languange)

fritssasia committed:
add documentation translation (indonesian languange)

fritssasia committed:
added info about the  folder in the  section

fritssasia committed:
added a dropdown menu to the original documentation, and also added info about the doc-translation folder in the structure projects section

parksb commented on README.md:
```diff
@@ -120,6 +129,7 @@ Done! You can change not only the landing page but any pages like this way. (You
   * `templates` - HTML files used as ejs template. `publish` script converts a markdown file to HTML based on templates under this directory.
 * `dist` - Files compiled by `build` script. `deploy` script deploys a website to GitHub pages based on this directory. Do not change the files under this directory directly.
 * `server` - Files compiled by `build` script. `start` script opens local server based on this directory. Do not change the files under this directory directly.
+* `doc-translation` - The folder contains documentation in several languages written in markdown format.
```
What do you think about placing `README-IND.md` at the root level?

fritssasia replied:
> What do you think about placing `README-IND.md` at the root level?

I don't know, what do you think? :"

parksb replied:
I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

fritssasia replied:
> I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

great idea, of course I'll do it

fritssasia replied:
@parksb I just moved the README-IND.md file to root, is this correct? What do you think? :"

I also changed the link in the README.md file (the link that will display the README-IND.md file)

fritssasia replied:
By the way, sorry if there is a mistake
I'm not very experienced in contributing to open source projects yet😅

parksb replied:
That's good 🙂 I left two comments suggesting little changes.

parksb replied:
That's good 🙂 I left two comments suggesting little changes.

parksb reviewed (commented):
Thank you :D

fritssasia committed:
fix errors on the link that will be directed to the `README-IND.md` file

Sorry it looks like I made a little mistake on the link which will show the README-IND.md file

before : https://github.com/parksb/handmade-blog/blob/parksb/doc-translation/README-IND.md
after    : https://github.com/parksb/handmade-blog/blob/master/doc-translation/README-IND.md

fritssasia committed:
move file README-IND.md to root

fritssasia committed:
README-IND.md documentation in Indonesian languange

fritssasia committed:
Merge branch 'fritssasia' of github.com:fritssasia/handmade-blog into fritssasia

fritssasia committed:
Update README.md

I changed the link which will display the `README-IND.md` file

parksb commented on README.md:
```diff
@@ -120,6 +129,7 @@ Done! You can change not only the landing page but any pages like this way. (You
   * `templates` - HTML files used as ejs template. `publish` script converts a markdown file to HTML based on templates under this directory.
 * `dist` - Files compiled by `build` script. `deploy` script deploys a website to GitHub pages based on this directory. Do not change the files under this directory directly.
 * `server` - Files compiled by `build` script. `start` script opens local server based on this directory. Do not change the files under this directory directly.
+* `doc-translation` - The folder contains documentation in several languages written in markdown format.
```
```suggestion
```

Please remove this line from README.md and README-IND.md

fritssasia replied:
@parksb Sorry for being sleepy I forgot to remove the doc-translation line, thanks for reminding me

I've finished removing the `doc-translation` line and changing the link which will display the `README-IND.md` file

parksb commented on README.md:
```diff
@@ -10,6 +10,15 @@
+<br>
+
+  * English (default)
+  * [Indonesian](https://github.com/parksb/handmade-blog/blob/master/README-IND.md)
```
```suggestion
  * [Indonesian](README-IND.md)
```

You can attach a path to file within the same project.

fritssasia committed:
remove the doc-translation line from the project structure

fritssasia committed:
Merge branch 'fritssasia' of github.com:fritssasia/handmade-blog into fritssasia

fritssasia committed:
change the link that will display the README-IND.md file

parksb commented:
LGTM. Thank you!

fritssasia commented:
By the way, does the `README-IND.md` file need to add a dropdown menu for the available language options? as in the `README.md` file

fritssasia commented:
> LGTM. Thank you!

You're welcome, thank you also for accepting me to contribute to this repository

parksb commented:
> By the way, does the `README-IND.md` file need to add a dropdown menu for the available language options? as in the `README.md` file

I don't think you have to, but if you have time, please send a new PR 😉
